src/functions/sendCandidateResults/domain/template-id.model.ts

```diff
@@ -0,0 +1,17 @@
+export type CorrespondenceMethod = 'Email' | 'Letter';
+
+export enum Language {
+  ENGLISH = 'english',
+  WELSH = 'welsh',
+  CYMRAEG = 'Cymraeg',
```

Comment:
Cymraeg valu capitalised whilst english and welsh aren't. needs to be consistent

Reply:
So there is different context to the values, the 'Cymraeg' value comes from the conductedLanguage of the test. The 'english' / 'welsh' is used as a prefix for the notify template language to use, I put all these values in the enum as they are related, should i separate these out?

Reply:
I think separate them out and rename so their purpose is clearer.